### ISSUE TITLE
keyboard shortcuts don't work with non English locale and more

### ISSUE THREAD

**Issue by wzdev-ci** [Warzone2100/old-trac-import#4847]
`resolution_closed` `type_bug`   |    by nikos_k

___

If I start Warzone, and I have already set my keyboard language to non English, then keyboard shortcuts don't work.

If, then, within the game, I change my keyboard's locale back to English, still the keyboard shortcuts don't work.

I need to (save the game, optionally) terminate Warzone, make sure my keyboard is set to English, and then start Warzone, for the shortcuts to function.

In other words, my keyboard must be set to English, before starting Warzone 2100, otherwise keyboard shortcuts won't work, in the game.

My system is...
```
   Warzone 2100 - Version: 3.2.3, Built:Apr 24 2017
   Mageia 6, 64bit
   keyboard: English / Greek
```

___
_Issue migrated from [trac:4847](https://developer.wz2100.net/ticket/4847) at 2022-04-16 13:33:28 -0700_
